tests/middleware/test_wsgi.py

```
@@ -0,0 +1,129 @@
+    response = client.post("/", json={"example": 123})
+    assert response.status_code == 200
+    assert response.text == '{"example": 123}'
+
+
+def test_wsgi_exception():
+    # Note that we're testing the WSGI app directly here.
+    # The HTTP protocol implementations would catch this error and return 500.
+    app = WSGIMiddleware(raise_exception)
+    client = TestClient(app)
+    with pytest.raises(RuntimeError):
+        response = client.get("/")
```

Ruff (0.11.9)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)